### PR TITLE
Minor input improvements

### PR DESCRIPTION
**Pull request by smharper**
Sorry to juggle the PRs.  This is the real one.  No more force pushes.

Two changes in this PR:  First, it allows you to add elements with this shortcut:
```python
# Old
b = openmc.Element('B')
material.add_element(b, 4e-3)
# New
material.add_element('B', 4e-3)
```
This has been possible for a long time with `add_nuclide`, and the appropriate code was already in place for `add_element`.  This case was just missed in the the `checkvalue`.

This PR also makes it so that settings.xml files are not read for plotting.  Now, only materials.xml and geometry.xml are needed for a plot.

### CODE REVIEW

**Comment by paulromano** [comment]
Since it is possible to show an outline of the entropy mesh on a plot, we still need to read in settings.xml.

**Comment by smharper** [comment]
Fixed.  Now the settings file will be read if it's there, but it's absence won't trigger an error.  If you try to plot an entropy mesh without a settings.xml file you'll get this error:
```
 ERROR: No entropy mesh for meshlines on plot 1
application called MPI_Abort(MPI_COMM_WORLD, -1) - process 0
[unset]: write_line error; fd=-1 buf=:cmd=abort exitcode=-1
:
system msg for write_line failure : Bad file descriptor
```

File: openmc/material.py [hunk 1]
```diff
@@ -401,7 +401,7 @@ def add_element(self, element, percent, percent_type='ao'):
                   'macroscopic data-set has already been added'.format(self._id)
             raise ValueError(msg)
 
-        if not isinstance(element, openmc.Element):
+        if not isinstance(element, (openmc.Element, str)):
```
Comment (paulromano) [inline openmc/material.py]: To be on the safe side, use `basestring` instead of `str`

**Comment by paulromano** [comment]
Looks good pending the one small fix

**Comment by smharper** [comment]
Fixed

**Pull request merged**